### PR TITLE
Add testing with molecule and refactor code

### PR DESCRIPTION
**Pull request by brucellino**
Major changes:

  - Add molecule with default scenario
    - test coverage only includes CEntOS7
    - test coverage only includes default file existence
  - Refactor tasks:
    - Include OS-specific variables before OS-specific tasks
    - Change syntax of task inclusion from include: to include_tasks:
    - Change failure message condition to check if the release version
      exists for the OS

### CODE REVIEW

File: vars/Debian9.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+release_url:
+  3: http://repository.egi.eu/sw/production/umd/3/debian/dists/squeeze/main/binary-amd64/umd-release-3.0.0_all.deb
+sources_dir: /etc/apt/sources.list.d
+nick: squeeze
```
Comment (orviz) [inline vars/Debian9.yml:5]: nick is not good

File: vars/Debian8.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+release_url:
+  3: http://repository.egi.eu/sw/production/umd/3/debian/dists/squeeze/main/binary-amd64/umd-release-3.0.0_all.deb
+sources_dir: /etc/apt/sources.list.d
+nick: squeeze
```
Comment (orviz) [inline vars/Debian8.yml:5]: nick is not good

File: vars/Debian7.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+release_url:
+  3: http://repository.egi.eu/sw/production/umd/3/debian/dists/squeeze/main/binary-amd64/umd-release-3.0.0_all.deb
+sources_dir: /etc/apt/sources.list.d
+nick: squeeze
```
Comment (orviz) [inline vars/Debian7.yml:5]: nick is not good

**Review by orviz — CHANGES_REQUESTED** [review]
re-check nick names

File: .travis.yml [hunk 4]
```diff
@@ -21,6 +21,7 @@ install:
 - pip install molecule docker
 - gem install inspec
 before_script:
+- ruby --versio:
```
Comment (gwarf) [inline .travis.yml]: Doesn't look right.

Comment (brucellino) [reply .travis.yml]: good catch! thanks.

**Pull request closed without merging**